packages/json/src/browser/json-client-contribution.ts

```diff
@@ -14,21 +14,17 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-import { injectable, inject } from 'inversify';
```

Comment:
@ndoschek can we minimize the changes here, they are unnecessary as far as I can tell.

Reply:
Sorry for that, updated it now.